### DISCUSSION TITLE
网站打不开

### DISCUSSION THREAD

**Discussion by tiebanluyu** [zai-org/ChatGLM3#713] [Q&A] [answered]
![image](https://github.com/THUDM/ChatGLM3/assets/108140288/7d399f82-c93c-431c-af72-c1d06137e035)
![image](https://github.com/THUDM/ChatGLM3/assets/108140288/3d1b2a6f-a523-41a3-998e-69143dc49831)

**Comment by tiebanluyu** [comment]
https://huggingface.co/THUDM/chatglm3-6b
打不开

OSError: We couldn't connect to 'https://huggingface.co' to load this file, couldn't find it in the cached files and it looks like THUDM/chatglm3-6b is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
要翻的。。。国内不能访问hf

**Comment by tiebanluyu** [comment]
大家都是中国人，何苦为难彼此呢？我肯定清华大学在科研方面的成果。但是，为什么要翻墙才能接受到你们的模型文件呢？为什么不把它放在国内呢？放在一个没有被墙的国外网站也行啊，为什么要放到一个已经被墙了的国外网站呢？似乎有一个链接可以从清华大学那边直接获取到模型文件，但是我找不到如何利用这些模型文件。


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年1月8日(周一) 下午3:22
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [THUDM/ChatGLM3] 网站打不开 (Discussion #713)




 
要翻的。。。国内不能访问hf
 
&mdash;

**Reply by zRzRzRzRzRzRzR** [reply]
尝试这里 https://modelscope.cn/models/ZhipuAI/chatglm3-6b/summary

**Comment by tiebanluyu** [comment]
依然不行



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年2月1日(周四) 中午12:48
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [THUDM/ChatGLM3] 网站打不开 (Discussion #713)




 
尝试这里 https://modelscope.cn/models/ZhipuAI/chatglm3-6b/summary

**Reply by zRzRzRzRzRzRzR** [reply]
这个还打不开就是你网络的问题了，这个是国内的网站

**Comment by tiebanluyu** [comment]
在我第二次说网站打不开之后，这个国内的网站下成功了。不过比较奇妙的是过个三五分钟，进度条突然前进30%多。不耐心的以为没下载。不过后面提示内存不足，没能看到模型回答问题。让两个g内存的电脑跑这个模型着实够呛。


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年2月1日(周四) 晚上7:28
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [THUDM/ChatGLM3] 网站打不开 (Discussion #713)




 
这个还打不开就是你网络的问题了，这个是国内的网站
 
&mdash;
